public/app/features/plugins/sql/components/visual-query-builder/SQLWhereRow.tsx

```diff
@@ -31,6 +32,18 @@ export function SQLWhereRow({ query, fields, onQueryChange, db }: WhereRowProps)
       config={{ fields: state.value || {} }}
       sql={query.sql!}
       onSqlChange={(val: SQLExpression) => {
+        const templateSrv = getTemplateSrv();
+        const templateVars = templateSrv.getVariables() as VariableWithMultiSupport[];
+
+        if (
+          templateVars.some((tv) => {
```

Comment:
More of a personal nit: maybe we can break this condition out into it's own function? Having 3 lines for a single condition is tough to read, in my view.